### DISCUSSION TITLE
`dagger bug` would open a new github issues with metadata filled in

### DISCUSSION THREAD

**Discussion by dolanor** [dagger/dagger#12053] [Ideas]
To have meaningful bug report, it would be good to have some subcommand that would automatically prefill OS+env information (OS type, version, dagger version, latest log, etc)
So the reporter can just add its own issue, without having to care about the whole context if it can get automatically.

**Comment by shykes** [comment]
I love this. A few follow-up thoughts:

* We can probably broaden this to also allow suggesting features, asking for help on a configuration, discussing a use case, reporting an issue with a specific package, search the knowledge base... A proposal to get us started: `dagger support` ?

* This feature works in tandem with a backend service. That service should be hosted by dagger.io, not github :) Of course we can still leverage github as much as we want, for example to create issues as you describe. But we should keep that logic on the backend service, not the client, to stay in control of the user experience.

* Perhaps each `dagger do` could record a unique job ID, such that `dagger support` can retrieve the job ID and metadata, pre-filling the support request with the details of the last failed job by default? Could be pretty killer.
